apps/web/components/App.tsx

```diff
@@ -114,36 +116,59 @@ export default function App({
               </div>
 
               <div className="mt-4 sm:mt-0 sm:text-right">
-                {!isLoading ? (
-                  isGlobal || installedAppCount > 0 ? (
-                    <div className="space-x-3">
+                {isLoading && <SkeletonButton width="24" height="10" />}
+
+                {!isLoading && multipleInstall && (
+                  <>
+                    {isGlobal || installedAppCount > 0 ? (
+                      <div className="space-x-3">
+                        <Button StartIcon={CheckIcon} color="secondary" disabled>
+                          {installedAppCount > 0
+                            ? t("active_install", { count: installedAppCount })
+                            : t("globally_install")}
+                        </Button>
+                        <InstallAppButton
+                          type={type}
+                          render={(buttonProps) => (
+                            <Button StartIcon={PlusIcon} data-testid="install-app-button" {...buttonProps}>
+                              {t("add_another")}
+                            </Button>
+                          )}
+                        />
+                      </div>
+                    ) : (
+                      <InstallAppButton
+                        type={type}
+                        render={(buttonProps) => (
+                          <Button data-testid="install-app-button" {...buttonProps}>
+                            {t("install_app")}
+                          </Button>
+                        )}
+                      />
+                    )}
+                  </>
+                )}
+
+                {!isLoading && !multipleInstall && (
+                  <>
+                    {installedAppCount > 0 && (
```

Comment:
Google Meet and Cal Video should be showing the "Installed" treatment and as they are global, with `installedAppCount === 0` they are not showing. This will get them that treatment, tested locally. 
```suggestion
                    {(isGlobal || installedAppCount > 0) && (
```